build/DependencyVersions.props

```diff
@@ -6,7 +6,8 @@
     <MsBuildPackagesVersion>0.1.0-preview-00028-160627</MsBuildPackagesVersion>
     <DependencyModelVersion>1.0.3</DependencyModelVersion>
     <PlatformAbstractionsVersion>1.0.3</PlatformAbstractionsVersion>
-    <NuGetVersion>4.0.0-rtm-2323</NuGetVersion>
+    <!-- non-official NuGet build taken from https://github.com/nuget/nuget.client/tree/release-4.1.0-netstandard2.0 to contain "2.0" TFMs -->
```

Comment:
I think we can remove the comment. @emgarten is getting the nuget changes merged into the 15.3 VS branch today